ci-operator/config/openshift-priv/installer/openshift-priv-installer-release-4.19.yaml

```diff
@@ -1255,7 +1258,13 @@ tests:
     post:
     - chain: cucushift-installer-rehearse-gcp-ipi-private-deprovision
     pre:
-    - chain: cucushift-installer-rehearse-gcp-ipi-private-provision
+    - ref: gcp-provision-vpc
```

Comment:
@jianlinliu we don't want to keep the `gcp-provision-minimal-permission` step here right?

Reply:
I did not see the job enabled mini permission on the var setting, so I removed it. So shall we add it back ?

Reply:
Or when we decide to enable mini permission, we can add it back on that time.

Reply:
ok to me

Reply:
Ah I just realized that it's a bit different for GCP to run the minimal-permission test, it requires the cluster profile to have a pre-configured SA with minimal-permission granted, so we don't have any minimal-permission test involved for GCP in the pre-submit.